config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json

```diff
@@ -3,22 +3,22 @@
         "5.0": {
             "status": "supported",
             "latest-revision": {
+                "revision": "10",
+                "release-date": "04 May 2023",
+                "cves": "CVE-2023-0361",
```

Comment:
I think you need to add additional CVEs `CVE-2023-24536,CVE-2023-24537,CVE-2023-24538` for golang vulnerability in this list.

Reply:
Updated